### ISSUE TITLE
Right way to install mods on a cluster setup

### ISSUE THREAD

**Issue by staddecx** [FezVrasta/ark-server-tools#1092]
Hello,

we are currently running 7 maps/instances on one dedicated server and I'm trying to figure out the right way to install mods.

We have about 17mods we want to use. I used to put all the IDs into the instances config files and then `arkmanager installmods @all`, but removing or installing another mods makes it a bit annoying in the long run, putting the ID into every single instance config.

I was thinking about to remove all mods which are currently listed in the instance configs (put comment on the `ark_GameModIds` line), then `arkmanager uninstallmod id1,id2,id3 @all` (all the IDs from the instance config) and then reinstall the mods I want with `arkmanager installmod id1,id2,id3 @all`.

Would this be the right way to have it cleaner and easier?

Thanks!

**Comment by klightspeed** [comment]
If a mod is shared between all of the instances, you should be able to use `arkmod_{modid}=enabled` in your `arkmanager.cfg`

Alternatively, you can use `arkmanager enablemod {modid} @all` to write this option to all of the instance configs.

**Comment by staddecx** [comment]
Oh I see, so its enough to put it into the `.arkmanager.cfg` instead of each instace since yeah we share all the mods with all the instances.

So I can comment the `ark_GameModIds={modids}` in each instance and put it into the `.arkmanager.cfg`. Yeah, thats much easier!

**Comment by klightspeed** [comment]
> Oh I see, so its enough to put it into the .arkmanager.cfg instead of each instace since yeah we share all the mods with all the instances.
> 
> So I can comment the ark_GameModIds={modids} in each instance and put it into the .arkmanager.cfg?

Yes.  As long as there are no options that need to be at the end of the server options string (options in `arkmanager.cfg` or `.arkmanager.cfg` will go onto the end of the server options string, usually in ASCIIbetical order), this should work.
